src/um/pdh.rs

```diff
@@ -1,9 +1,14 @@
 // Copyright © 2016, Klavs Madsen
 // Licensed under the MIT License <LICENSE.md>
+
 //! Common Performance Data Helper definitions
-use shared::minwindef::{DWORD};
-use shared::wtypesbase::{DOUBLE};
-use um::winnt::{LPWSTR, LPCWSTR, LPSTR, LPCSTR, LONGLONG, LONG, HANDLE};
+
+use shared::basetsd::DWORD_PTR;
+use shared::guiddef::GUID;
+use shared::minwindef::{BOOL, DWORD, FILETIME, LPDWORD, UCHAR};
+use shared::windef::HWND;
+use shared::wtypesbase::DOUBLE;
+use um::winnt::{LPWSTR, LPCWSTR, LPSTR, LPCSTR, LONGLONG, LONG, HANDLE, BOOLEAN};
```

Comment:
Don't forget to update `build.rs` to reflect the changes to header dependencies.